### PR TITLE
[ResourceAction] LPS-123071 Clean up default portlet resource actions population

### PR DESCRIPTION
**Pull request by dantewang**

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test:sf

**Comment by dantewang** [comment]
ci:test:relevant

**Comment by dantewang** [comment]
ci:test

File: portal-impl/src/com/liferay/portal/security/permission/ResourceActionsImpl.java [hunk 1]
```diff
@@ -903,9 +903,7 @@ else if (StringUtil.equalsIgnoreCase(
 		synchronized (this) {
 			portletActions = _getPortletMimeTypeActions(name, portlet);
 
-			if (!name.equals(PortletKeys.PORTAL)) {
-				_checkPortletActions(portlet, portletActions);
-			}
+			_checkPortletActions(portlet, portletActions);
```
Comment (tinatian) [inline portal-impl/src/com/liferay/portal/security/permission/ResourceActionsImpl.java:889]: @dantewang Could you please describe the reason in more details? Why it won't be invoked?

Tina.

**Comment by tinatian** [comment]
See https://github.com/shuyangzhou/liferay-portal/pull/9603

**Pull request closed without merging**